client/Gruntfile.js

```diff
@@ -67,7 +68,8 @@ module.exports = function(grunt) {
             "**",
             "!js/**/*.js",
             "lib/js/*.js",
-            "lib/js/locale/*.js"
+            "lib/js/locale/*.js",
+            "secret.html"
```

Comment:
@joeman65 @alessiofranceschini : this seems to be code of the last year project that is very specific to ANAC and it has been evaluated to keep unmerged. Why are you proposing it again here?

Would you please remove this change?